runtime/guard-jwt/src/main/java/io/aklivity/zilla/runtime/guard/jwt/internal/JwtGuardHandler.java

```diff
@@ -97,14 +86,20 @@ else if (options.keysURL.isPresent())
 
                 String keysText = readURL.apply(URI.create(options.keysURL.get()).toURL());
                 JwtKeySetConfig jwks = jsonb.fromJson(keysText, JwtKeySetConfig.class);
-                List<JwtKeyConfig> keysConfig = jwks.keys;
-                resolvedKeys = getKeys(keysConfig);
+                keysConfig = jwks.keys;
             }
             catch (MalformedURLException | JsonbException ex)
             {
                 rethrowUnchecked(ex);
             }
         }
+
+        Map<String, JsonWebKey> resolvedKeys = new HashMap<>();
+        if (keysConfig != null)
+        {
+            resolvedKeys = getKeys(keysConfig);
```

Comment:
I was requesting you inline `getKeys` method body here, no need for separate method any more.